src/app/plugins/platform/Docker/dockerfile/composer.ts

```diff
@@ -0,0 +1,15 @@
+/**
+ * Currently-supported Composer version.
+ */
+export const composerTag = '1.7';
+
+/**
+ * The name of the Composer image to use for projects.
+ */
+export const composerImage = 'composer';
+
+/**
+ * The full `<image>:<tag>` string for this image; useful for `docker run` or the `image:`
+ * field in Docker Compose.
+ */
+export const composer = `${composerImage}:${composerTag}`;
```

Comment:
Nice touch splitting these out and maintaining them separately. That'll be easier to keep track of and update.